### PR TITLE
Main page prototype

### PR DESCRIPTION
**Pull request by nyotaL**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,7 +17,9 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Avoid fruits and nuts. You are what you eat', ' Chop your own wood and it will warm you twice', 
+      ' Life is far too important a thing ever to talk seriously about', 
+      'Your future depends on your dreams. So go to sleep', 'I know that you believe you understand what you think I said, but I’m not sure you realize that what you heard is not what I meant.'];
```
Comment (jeadorf) [inline portfolio/src/main/webapp/script.js]: It's easier to read and review changes when putting each list item on its separate line.

Also please see https://google.github.io/styleguide/jsguide.html#formatting-column-limit.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,13 +7,31 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    <p>Click <a href="Kite_gallery.html">here</a> to see my kite gallery!</p>
+        <div id="content">
+        <h1>Katya's Portfolio</h1>
+        <text id = "intro"> Hi! My name is Katya...</text> 
+
+        <div class="display">
```
Comment (jeadorf) [inline portfolio/src/main/webapp/index.html]: https://google.github.io/styleguide/htmlcssguide.html
asks to consistently indent blocks in HTML with 2 spaces

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,3 +1,22 @@
+a {
```
Comment (jeadorf) [inline portfolio/src/main/webapp/style.css:1]: https://google.github.io/styleguide/htmlcssguide.html
also here: 2 spaces indentation

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -48,3 +68,97 @@
...
+    background: rgb(250, 212, 227);
+}
+
+#MKG span {
+    background: rgb(214, 252, 205);
+}
+
+.HOVER:hover span {
+	width: calc(var(--width) * 2.25);
+	padding-top: calc(var(--width) * 2.25);
```
Comment (jeadorf) [inline portfolio/src/main/webapp/style.css]: Interesting. I did not know about https://www.w3schools.com/css/css3_variables.asp

**Review by jeadorf — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -7,13 +7,31 @@
...
-    <div id="content">
-      <h1>Katya's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    <p>Click <a href="Kite_gallery.html">here</a> to see my kite gallery!</p>
+        <div id="content">
+        <h1>Katya's Portfolio</h1>
+        <text id = "intro"> Hi! My name is Katya...</text>
```
Comment (AndreMB0) [inline portfolio/src/main/webapp/index.html]: please remove trailing whitespaces on text and on attributes as well
https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,3 +1,22 @@
+a {
+    text-decoration: none;
+}
+
+button {
+    background: whitesmoke;
+    width: 70px;
+    height: 70px;
+    border: thin solid rgba(0, 0, 0, 0.726);
+}
```
Comment (AndreMB0) [inline portfolio/src/main/webapp/style.css]: Put declarations in alphabetical order.
Also below
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -7,13 +7,31 @@
...
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    <p>Click <a href="Kite_gallery.html">here</a> to see my kite gallery!</p>
+        <div id="content">
+        <h1>Katya's Portfolio</h1>
+        <text id = "intro"> Hi! My name is Katya...</text> 
+
+        <div class="display">
+            <a href = "TIL.html" class="HOVER" id = "TIL">
```
Comment (AndreMB0) [inline portfolio/src/main/webapp/index.html]: Please use only lowercase, also below in in css
https://google.github.io/styleguide/htmlcssguide.html#Capitalization

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -7,13 +7,31 @@
...
+                <span></span>
```
Comment (AndreMB0) [inline portfolio/src/main/webapp/index.html]: You may want to try to apply the same style outcome without using empty span

File: portfolio/src/main/webapp/kite_gallery.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html id = "gallery">
+  <head>
+    <meta charset="UTF-8">
+    <title>Kite Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="gallery_content">
+      <h1 id = "g_title">My Kite Image Gallery</h1>
```
Comment (jeadorf) [inline portfolio/src/main/webapp/kite_gallery.html:10]: minor comment: there are a few more cases of whitespaces around HTML attributes.

File: portfolio/src/main/webapp/kite_gallery.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+      <div class="image-thumbnails">
```
Comment (jeadorf) [inline portfolio/src/main/webapp/kite_gallery.html:11]: that's a great example of how to implement https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns
good

**Review by jeadorf — APPROVED** [review]
Do the empty spans in the HTML serve a purpose?

**Pull request merged**